### PR TITLE
Feature items index public

### PR DESCRIPTION
**Pull request by stephan-dev**

### CODE REVIEW

File: app/assets/stylesheets/home.scss [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	height:40px;
+	width:100px;
+	text-align: center;
+	vertical-align: middle;
+}
+
+img {
+	height: 150px;
+	vertical-align: middle;
+}
```
Comment (Crysicia) [inline app/assets/stylesheets/home.scss:18]: Use 2 spaces indentation

File: app/views/items/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+		<td>
+			<%= item.original_price.round(2) %> €<br>
+		</td>
+		<td>
+		    <img src="<%=item.photo%>"><br><br>
+		</td>
+	</tr>
+<% end %>
+
+</table>
```
Comment (Crysicia) [inline app/views/items/index.html.erb:22]: Use 2 spaces indentation

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  root 'items#index'
+  get 'items/index'
+  get 'items/show'
```
Comment (Crysicia) [inline config/routes.rb:6]: Use resource instead of GET

**Review by Crysicia — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/home.scss (hunk 1 above)
Comment (stephan-dev) [reply app/assets/stylesheets/home.scss:18]: ok

**Comment by stephan-dev** [comment]
PR avortée, trop de conflits.

**Pull request closed without merging**